source/_patterns/02-molecules/blocks/timeline-item/timeline-item.twig

```diff
@@ -1,9 +1,15 @@
-<div class="jcc-timeline-item">
+<li class="jcc-timeline-item">
   <h3 class="jcc-timeline-item__title">{{ title }}</h3>
   <div class="jcc-timeline-item__excerpt">{{ excerpt }}</div>
 
-  {% if read_more_link %}
-  <a href="{{ url }}" class="jcc-timeline-item__trigger"{% if url_attributes %}{{ url_attributes }}{% endif %}>{{ read_more_link }}</a>
-  {% endif %}
+  {% if read_more.url %}
+    <a href="{{ read_more.url }}" class="jcc-timeline-item__trigger"{% if url_attributes %}{{ url_attributes }}{% endif %} title="{{ read_more.label }}" target="_blank">{{ read_more.label }}</a>
+  {% elseif read_more.id %}
+    {% include '@organisms/sections/full-modal/full-modal.twig' with {
+      modal_id: read_more.id,
+      button_label: read_more.label
+    } %}
+  {% else %}
```

Comment:
This works!  👏 

Just need to use the `read_more.url` attributes, and make sure it's an attributes object. Adding this line above the `<a>` will work.

```twig
{% set url_attributes = read_more.url_attributes ? read_more.url_attributes : create_attribute() %}
```
Then, in the link you can just print it without the if statement.

```twig
<a href="{{ read_more.url }}" class="jcc-timeline-item__trigger" title="{{ read_more.label }}"{{ url_attributes }}>{{ read_more.label }}</a>
```

BTW, I removed `target="_blank`  that might be coming through `url_attributes`. Do you really need it? If so, I can show you how to make that happen. Let me know.

Also, do you need the `title` attribute?  I'm guessing you might need it for the modal JS title, but if not, we should remove it. It's not good from an accessibility standpoint, when the link text and title are the same.

Reply:
Thanks @jacine. 

`target="_blank"` is a best practice when we are opening external links. I assume the use case for these anchor tags are exactly that. That's why I used it. I think we should keep them. 

I always thought that the `title` attribute was both an accessibility and SEO best practice. Your comment about repeating it makes sense, so I removed it.

Reply:
@claudioccm Thanks!  Yeah, I agree with you that external links should open in a new window.  I just don't think that will be the case here.  We don't have a URL field for this, and even if we did, AFAIK, this likely won't be an external link.

However, this is how you _could_ achieve an external link here, while still allowing Drupal an out, which would be necessary if this ever did become a link field (I don't think it will), because it's entirely possible that content editors would be able to choose the desired target themselves.  This is a big part of the reason we print `url_attributes` to make sure we don't break this sort of functionality.

1. First we would need an option. I used `target: blank` in `timeline-item.yml`

    ```diff
        read_more:
          label: "Strategy Details"
          url: "#"
    +      target: 'blank'
    ```

2. Then, use `setAttribute` the target to add them based on above.  Note:  You should always include `rel="noopener"` anywhere you use `target="_blank"`. See [this article](https://mathiasbynens.github.io/rel-noopener/).

    ```diff
      {% if read_more.url %}
        {% set url_attributes = read_more.url_attributes ? read_more.url_attributes : create_attribute() %}
    +    {% if read_more.target == 'blank' %}
    +      {% set url_attributes = url_attributes
    +        .setAttribute('target', '_blank')
    +        .setAttribute('rel', 'noopener' )
    +      %}
    +    {% endif %}
        <a href="{{ read_more.url }}" class="jcc-timeline-item__trigger" {{ url_attributes }}>{{ read_more.label }}</a>
    ```

This would result in the output you want, and would also allow Drupal to set the target to `false` and pass through the attributes from the API.